releasenotes/notes/iterative-phase-estimation-bugfix-b676ffc23cea8251.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixed a bug where the iterative phase estimation algorithm was generating the wrong circuit, causing the algorithm to fail for simple cases. Refer to `#9280 https://github.com/Qiskit/qiskit-terra/issues/9280` for more details.
```

Comment:
```suggestion
    Fixed a bug where :class:`.IterativePhaseEstimation` was generating the wrong circuit, causing the
    algorithm to fail for simple cases. Fixed `#9280 https://github.com/Qiskit/qiskit-terra/issues/9280`__.
```

Reply:
Note that there is no newline at the end of the release note file. I'm not sure if the suggested change of @Cryoris also fixed this. But it should be fixed.